JamfSync/Model/DistributionPoint.swift

```diff
@@ -322,7 +322,9 @@ class DistributionPoint: Identifiable {
         if willDownloadFiles {
             downloadMultiple = 2
         }
+        print("[copyFilesToDst] Synchronizing \(filesToSync.count) to \(dstDp.name)")
```

Comment:
These two print statements should either be debug log messages or removed. A good rule of thumb for keeping it as a debug log message or removing it are asking yourself how likely will it help with future problems that may be hard to reproduce by ourselves. If the answer is "not likely", delete it. You can always temporarily add log messages when debugging a problem that you can recreate. If you keep too many log messages around, it just results in a lot of noise.

Reply:
removed